app/scss/themes/core/_variables_dark.scss

```diff
@@ -1,3 +1,7 @@
+/**
+ * TODO .... These are not correct ... just samples.
+ */
+ 
 // Colors
 $primary: #528100;
 $complimentary: #7C3FE8;
```

Comment:
I went ahead and created the files for light and dark, however the color variables are intended to be updated to match the themes provided by Nina.